Framework/Algorithms/src/ConvertSpectrumAxis2.cpp

```diff
@@ -143,8 +145,6 @@ void ConvertSpectrumAxis2::createElasticQMap(API::Progress &progress,
                                              const std::string &targetUnit,
                                              API::MatrixWorkspace_sptr &inputWS,
                                              size_t nHist) {
```

Comment:
I think the `nHist` argument can be removed, and `spectrumInfo.size()` can be used instead.